tests/handlers/records-read.spec.ts

```diff
@@ -1614,6 +1917,49 @@ export function testRecordsReadHandler(): void {
       });
     });
 
+    describe('createFilter()', async () => {
+      it('should not throw if only `recordId` is set', async () => {
+        const filter = RecordsReadHandler.createFilter({
+          interface        : DwnInterfaceName.Records,
+          method           : DwnMethodName.Read,
+          messageTimestamp : '2023-08-19T00:00:00.000000Z',
+
+          recordId: 'some-id'
+        });
+
+        expect(filter['recordId']).to.equal('some-id');
+      });
+
+      it('should not throw if `protocol` and `protocolPath` are set', async () => {
```

Comment:
Same as https://github.com/TBD54566975/dwn-sdk-js/pull/470/files#r1299587478
Sanity: This method never throws. Why test that it doesn't throw under these conditions?

Reply:
Henry made an update removing the explicitly thrown message here since JSON Schema will throw an exception elsewhere.

I'll update these tests to reflect that better.